tools/report-converter/codechecker_report_converter/sparse/output_parser.py

```diff
@@ -0,0 +1,106 @@
+# -------------------------------------------------------------------------
+#
+#  Part of the CodeChecker project, under the Apache License v2.0 with
+#  LLVM Exceptions. See LICENSE for license information.
+#  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# -------------------------------------------------------------------------
+
+import logging
+import os
+import re
+
+from ..output_parser import BaseParser, Message, Event
+LOG = logging.getLogger('ReportConverter')
+
+
+class SparseParser(BaseParser):
+    """
+    Parser for Sparse Output
+    """
+
+    def __init__(self, analyzer_result):
+        super(SparseParser, self).__init__()
+
+        self.analyzer_result = analyzer_result
+
+        self.message_line_re = re.compile(
+            # File path followed by a ':'.
+            r'^(?P<path>[\S ]+?):'
+            # Line number followed by a ':'.
+            r'(?P<line>\d+?):'
+            # Column number followed by a ':'.
+            r'(?P<column>\d+?):'
+            # Message.
+            r'(?P<message>[\S \t]+)\s*')
+
+        self.multi_line_message_re = re.compile(
+            r'^(?P<path>[\S ]+?):'
+            r'(?P<message>[\S \t]+)\s*'
+        )
```

Comment:
We don't need this regex if we don't want the lines of type:
`files/sample.c: note: Message`

Reply:
Yes, we can remove it.